op-e2e/system_test.go

```diff
@@ -10,9 +10,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ethereum-optimism/optimism/op-node/eth"
```

Comment:
your IDE is still doing weird go import things. It's maybe worth figuring out an auto formatter for imports (gofmt only groups imports inside the block)